client/src/Movies/MovieList.js

```diff
@@ -35,6 +35,7 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
```

Comment:
Good! So to review, that's the same as:
`const title = movie.title;`
`const director = movie.director`
etc...